### PR TITLE
System canisters doc

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/production/system-canisters.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Overview
+
+System canisters are canisters that provide a necessary function to the ICP network. These canisters are controlled by the NNS and are upgraded via proposals. These canisters have static canister IDs that projects can reference. One common example of a system canister that is referenced by projects is the Internet Identity canister. 
+
+## System canisters
+
+| System canister | Canister ID |
+|-----------------|-------------|
+| NNS registry        | rwlgt-iiaaa-aaaaa-aaaaa-cai |
```
Comment (jennifertrin) [inline docs/developer-docs/production/system-canisters.mdx:11]: I suggest linking to any docs on the topics unless we feel that developers will not reference this doc until they have a pretty thorough understanding of ICP

**Review by dfx-json — APPROVED** [review]

File: docs/developer-docs/production/system-canisters.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+| ICP ledger archive | qsgjb-riaaa-aaaaa-aaaga-cai |
+| ICP ledger archive | qjdve-lqaaa-aaaaa-aaaeq-cai |
+| NNS root | r7inp-6aaaa-aaaaa-aaabq-cai |
+| Cycles minting | rkp4c-7iaaa-aaaaa-aaaca-cai |
+| NNS lifeline | rno2w-sqaaa-aaaaa-aaacq-cai |
+| Genesis token | renrk-eyaaa-aaaaa-aaada-cai |
+| SNS wasm | qaa6y-5yaaa-aaaaa-aaafa-cai |
+| Internet Identity | rdmx6-jaaaa-aaaaa-aaadq-cai |
+| NNS UI | qoctq-giaaa-aaaaa-aaaea-cai |
+| Motoko playground | m7sm4-2iaaa-aaaab-qabra-cai |
```
Comment (dsarlis) [inline docs/developer-docs/production/system-canisters.mdx:24]: This is debatable. Motoko playground is a canister developed by DFINITY but it's not controlled by NNS nor does it provide critical functionality (as other canisters listed here). It's certainly a useful canister but I think it's best to not include it in this list.

We could consider adding the bitcoin (mainnet: `ghsi2-tqaaa-aaaan-aaaca-cai` and testnet: `g4xu7-jiaaa-aaaan-aaaaq-cai`) canisters. I think those two fit the bill -- they provide functionality that extends the capabilities of ICP and are controlled by the NNS.

File: docs/developer-docs/production/system-canisters.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# System canisters
+
+## Overview
+
+System canisters are canisters that provide a necessary function to the ICP network. These canisters are controlled by the NNS and are upgraded via proposals. These canisters have static canister IDs that projects can reference. One common example of a system canister that is referenced by projects is the Internet Identity canister.
```
Comment (dsarlis) [inline docs/developer-docs/production/system-canisters.mdx:5]:
```suggestion
System canisters are canisters that provide a necessary function to the ICP network. These canisters are controlled by the NNS and are upgraded via NNS proposals. These canisters have static canister IDs that projects can reference. One common example of a system canister that is referenced by projects is the Internet Identity canister. 
```
just to ensure no confusion with SNS proposals.

File: docs/developer-docs/production/system-canisters.mdx (hunk 2 above)
Comment (domwoe) [reply docs/developer-docs/production/system-canisters.mdx:24]: We should also add the Exchange Rate Canister. I'm also on the fence about adding ckBTC and ckETH canisters here, or if this information should be only in the dedicated section of the docs.

File: docs/developer-docs/production/system-canisters.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (jessiemongeon1) [reply docs/developer-docs/production/system-canisters.mdx]: Added links to the NNS docs

[the remark this answers is not in the document]
Comment (jessiemongeon1) [reply docs/developer-docs/production/system-canisters.mdx]: Added info about why/how a dev would reference a system canister

File: docs/developer-docs/production/system-canisters.mdx (hunk 2 above)
Comment (jessiemongeon1) [reply docs/developer-docs/production/system-canisters.mdx:24]: Removed Motoko playground and added the Bitcoin testnet/mainnet. For ckBTC/ckETH, I think they would fit best on the pages dedicated to each.

**Pull request merged**